e2e/setupTests.js

```diff
@@ -1,5 +1,5 @@
 const { toMatchImageSnapshot } = require('jest-image-snapshot');
 
-jest.setTimeout(60000);
+jest.setTimeout(120000);
```

Comment:
this is for what reason?

Reply:
It's because too many services are involved and right after they're up, they take a bit time to respond to requests.

We can probably set it lower even back to 1min but this isn't really important since it is just MAX allowed time per test case.